test/mason/publish-tests/publishHelp.good

```diff
@@ -8,5 +8,6 @@ Options:
     -h, --help                   Display this message
     --dry-run                    Check to see if package is ready to be published
     --no-update                  Prevents registrys from being updated when a package is published.
```

Comment:
typo nit: Could you do a grep for registrys and replace with registries in mason files / testing?